tools/amd_build/build_caffe2_amd.py

```diff
@@ -26,7 +26,6 @@
 ignores = [
     "caffe2/operators/depthwise_3x3_conv_op.cu",
     "caffe2/operators/depthwise_3x3_conv_op_cudnn.cu",
-    "caffe2/operators/top_k.cu",
     "caffe2/operators/top_k_radix_selection.cuh",
```

Comment:
Purely out of interest: aren't these close enough to TopK to just get enabled as well?